Products/Zuul/routers/manufacturers.py

```diff
@@ -154,11 +154,10 @@ def moveProduct(self, moveFrom, moveTarget, ids):
             "Scheduled job to move %d product(s)." % len(ids)
         )
         
-    def returnTree(self, id):
+    def returnManufacturerList(self):
```

Comment:
If you're going to change the name of this method, let's also change it to `getManufacturerList` so that it follows common naming convention for getter functions.